packages/dashboard/scripts/setup/get-icons.js

```diff
@@ -4,10 +4,8 @@ const fs = require('fs');
 const chalk = require('chalk');
 const path = require('path');
 
-const curWorkingDir = process.cwd();
-let file = curWorkingDir.includes('dashboard')
-  ? './.resources.json'
-  : './dashboard/.resources.json';
+const basePath = __dirname.slice(0, __dirname.length - 'scripts/setup'.length);
```

Comment:
nit: This might not work in windows (not that we support it), using the `path` api, or using `..` to go back one level is more cross platform friendly.